### ISSUE TITLE
Hi,I am using DBN for regression

### ISSUE THREAD

**Issue by BangpengGao** [matrachma/Deep-Belief-Network-for-Regression#1]
I used your code,but the result was same in test data.May you help me?

**Comment by terapikode** [comment]
@BangpengGao maybe your model was overfitting, try to tune you hyper-parameter such as the number hidden layer/unit, learning rate, etc.

**Comment by BangpengGao** [comment]
Thank you.Could you give me the test data?

**Comment by wangyexiang** [comment]
Thank you for your awesome code. Can this code be converted to tensorflow form?

**Comment by caozhanxu** [comment]
Thank you . Could you upload the train data and the test data  ? or upload some samples.  Thank you !

**Comment by yanmou8096** [comment]
hello Could you give me your dataset ? thank you!

**Comment by yanmou8096** [comment]
@Hikmat Rachmatia could you upload your datasets? Thank you very much!
